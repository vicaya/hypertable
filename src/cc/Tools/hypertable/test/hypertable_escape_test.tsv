\\newline\ntab\tslash\\	TestColumnFamily:\\newline\ntab\tslash\\	hello, world!
