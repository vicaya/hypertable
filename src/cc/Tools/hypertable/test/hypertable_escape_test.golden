#row	column	value
\newline
tab	slash\	TestColumnFamily:\newline
tab	slash\	hello, world!
